src/components/tools/requests/Details.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @author sriram
+ *
+ *
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+import { useQuery } from "react-query";
+
+import ids from "../ids";
+import {
+    ADMIN_TOOL_REQUEST_DETAILS_QUERY_KEY,
+    getAdminToolRequestDetails,
+} from "serviceFacades/tools";
+
+import { build, FormTextField } from "@cyverse-de/ui-lib";
+
+import DEDialog from "components/utils/DEDialog";
+import ErrorTypographyWithDialog from "components/utils/error/ErrorTypographyWithDialog";
+
+import { Field, Form, Formik } from "formik";
+import { Button } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+
+function DetailsDialog(props) {
+    const { open, parentId, requestId, onClose } = props;
+    const { t } = useTranslation("tools");
+    const { t: i18nCommon } = useTranslation("common");
+    const [details, setDetails] = React.useState({
+        name: "",
+        description: "",
+        test_data_path: "",
+        source_url: "",
+        version: "",
+        documentation_url: "",
+        cmd_line: "",
+    });
+
+    const { isFetching: isDetailsFetching, error: detailsFetchError } =
+        useQuery({
+            queryKey: [ADMIN_TOOL_REQUEST_DETAILS_QUERY_KEY, { id: requestId }],
+            queryFn: getAdminToolRequestDetails,
+            config: {
+                enabled: requestId && open,
+                onSuccess: (resp) => {
+                    setDetails(resp);
+                },
+            },
+        });
+
+    return (
+        <Formik
+            initialValues={details}
+            enableReinitialize={true}
+        >
+            {({ handleSubmit, values }) => {
+                return (
+                    <Form>
+                        <DEDialog
+                            open={open}
+                            fullWidth={true}
+                            onClose={onClose}
+                            id={parentId}
+                            title="Details"
+                            actions={
+                                    <Button
+                                        id={build(parentId, ids.BUTTONS.OK)}
+                                        type="submit"
+                                        color="primary"
+                                    onClick={onClose}
+                                    >
+                                        {i18nCommon("ok")}
+                                </Button>
```

Comment:
Looks like a weird formatting issue here (I'm surprised the `prettier` check didn't throw an error about it).

Reply:
Weird. When I format with VS Code extension prettier, it stays like this. But when I use our `npm` task, this was fixed. 😕